content/_includes/post.macros.njk

```diff
@@ -268,6 +279,7 @@ params:
             ) }}
```

Comment:
When there's more than one date here ("past" or "upcoming"), it's really not clear what year each date is. In this example, the dates displayed are actually from 2015 and 2018:

![Screen Shot 2021-04-15 at 9 17 56 AM](https://user-images.githubusercontent.com/552316/114875823-afe6ea00-9dcb-11eb-9cc0-0a4c8c86d9f6.png)